storybookjs/storybook
# Addon-docs: Add opt-in table of contents

Closes N/A

## What I did

Added opt-in TOC to docs page to address the sidebar navigation issues caused by the 7.0 docs changes.

This is the "built-in" version of https://github.com/storybookjs/addon-toc/pull/1

Added:
- [x] `docs.toc` parameter
- [x] Example stories to demonstrate the key configuration parameters

## How to test

1. Spin up a sandbox.
2. Browse to the added `addon-docs/toc` stories which demonstrate different configurations
3. Add `docs.toc = { }` parameter to `.storybook/preview.js` to see how it looks for all the pages
